python/ray/data/_internal/execution/interfaces/op_runtime_metrics.py

```diff
@@ -12,266 +12,289 @@
+        description="Byte size of input blocks received by operator.",
+        metrics_group="inputs",
     )
-    num_task_inputs_processed: int = field(
+    num_task_inputs_processed: int = metricfield(
         default=0,
-        metadata={
-            "description": (
-                "Number of input blocks that operator's tasks "
-                "have finished processing."
-            ),
-            "metrics_group": "inputs",
-            "map_only": True,
-        },
-    )
-    bytes_task_inputs_processed: int = field(
+        description=(
+            "Number of input blocks that operator's tasks have finished processing."
+        ),
+        metrics_group="inputs",
+        map_only=True,
+    )
+    bytes_task_inputs_processed: int = metricfield(
         default=0,
-        metadata={
-            "description": (
-                "Byte size of input blocks that operator's tasks "
-                "have finished processing."
-            ),
-            "metrics_group": "inputs",
-            "map_only": True,
-        },
-    )
-    bytes_inputs_of_submitted_tasks: int = field(
+        description=(
+            "Byte size of input blocks that operator's tasks have finished processing."
+        ),
+        metrics_group="inputs",
+        map_only=True,
+    )
+    bytes_inputs_of_submitted_tasks: int = metricfield(
         default=0,
-        metadata={
-            "description": "Byte size of input blocks passed to submitted tasks.",
-            "metrics_group": "inputs",
-            "map_only": True,
-        },
+        description="Byte size of input blocks passed to submitted tasks.",
+        metrics_group="inputs",
+        map_only=True,
     )
 
     # === Outputs-related metrics ===
-    num_task_outputs_generated: int = field(
+    num_task_outputs_generated: int = metricfield(
         default=0,
-        metadata={
-            "description": "Number of output blocks generated by tasks.",
-            "metrics_group": "outputs",
-            "map_only": True,
-        },
+        description="Number of output blocks generated by tasks.",
+        metrics_group="outputs",
+        map_only=True,
     )
-    bytes_task_outputs_generated: int = field(
+    bytes_task_outputs_generated: int = metricfield(
         default=0,
-        metadata={
-            "description": "Byte size of output blocks generated by tasks.",
-            "metrics_group": "outputs",
-            "map_only": True,
-        },
+        description="Byte size of output blocks generated by tasks.",
+        metrics_group="outputs",
+        map_only=True,
     )
-    rows_task_outputs_generated: int = field(
+    rows_task_outputs_generated: int = metricfield(
         default=0,
-        metadata={
-            "description": ("Number of output rows generated by tasks."),
-            "metrics_group": "outputs",
-            "map_only": True,
-        },
+        description=("Number of output rows generated by tasks."),
```

Comment:
nit: parentheses not needed
```suggestion
        description="Number of output rows generated by tasks.",
```